### PR TITLE
Add simplified mappings "UploadConversionAdjustment" action

### PR DESCRIPTION
**Pull request by marinhero**
This PR adds a new action for UploadConversionAdjustment. The action implements the Sync Mode for Simplified Mappings.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  getConversionActionDynamicData,
+  isHashedInformation
+} from '../functions'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { PartialErrorResponse, ConversionAdjustmentRequestObjectInterface, UserIdentifierInterface } from '../types'
+import { ModifiedResponse } from '@segment/actions-core'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Upload Conversion Adjustment (Simplified)',
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts:17]: `(Simplified)`

File: packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+  dynamicFields: {
+    conversion_action: async (
+      request: RequestClient,
+      { settings, auth, features, statsContext }
+    ): Promise<DynamicFieldResponse> => {
+      return getConversionActionDynamicData(request, settings, auth, features, statsContext)
+    }
+  },
+  perform: async (request, { settings, payload, features, statsContext, syncMode }) => {
+    if (syncMode === 'add') {
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts:243]: `syncMode`

File: packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+} from '../functions'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import { PartialErrorResponse, ConversionAdjustmentRequestObjectInterface, UserIdentifierInterface } from '../types'
+import { ModifiedResponse } from '@segment/actions-core'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Upload Conversion Adjustment (Simplified)',
+  description: 'Upload a conversion adjustment to the Google Ads API.',
+  syncMode: {
```
Comment (marinhero) [inline packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts:19]: `syncMode`

File: packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+    description: 'Define how the records from your destination will be synced.',
+    label: 'How to sync records',
+    default: 'add',
+    choices: [{ label: 'Insert Records', value: 'add' }]
+  },
+  fields: {
+    conversion_action: {
+      label: 'Conversion Action ID',
+      description: 'The ID of the conversion action associated with this conversion.',
+      type: 'number',
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts:35]: was this also a field where the type should be updated to 'string' or no?

**Review by nick-Ag — APPROVED** [review]

File: packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts (hunk 4 above)
Comment (marinhero) [reply packages/destination-actions/src/destinations/google-enhanced-conversions/uploadConversionAdjustment2/index.ts:35]: I'm not touching the actions at the moment I'm just leaving them the way they were but with Sync Mode enabled.

**Review by maryamsharif — APPROVED** [review]

**Review by nick-Ag — APPROVED** [review]

**Pull request merged**